### DISCUSSION TITLE
馬斯克預估：2050 年家家戶戶可能都有人形機器人

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#583] [科技新知]
馬斯克（Elon Musk）在《TED》最新上傳的[採訪影片](https://www.youtube.com/watch?v=YRvf00NooN8&t=2250s)中強調，他花了一段時間了解到，想要解決自駕問題，就必須先解決真實世界的 AI 問題。一旦為一輛車子解決了真實世界的 AI 問題時，那麼它就會變成一輛有 4 個輪子的機器人，同樣的，擁有雙腿的機器人也會有所進展。儘管在改善 AI 的道路上，仍然充滿許多重大阻礙，但馬斯克認為約莫 2050 年，大部分的家庭將擁有自己專屬的家庭機器人。

https://technews.tw/2022/04/20/robot-at-home/

**Comment by yenwei12138** [comment]
馬斯克真的很有才有想法
還有你的採訪影片怎麼弄得，太厲害了吧~

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期待未來的發展。

**Comment by sys511613** [comment]
如果人造機器人要出現了，那跨越恐怖谷也是一個需要挑戰的問題。
現在也有很多團隊(像是[Disney Research](https://www.disneyresearch.com/)、[Engineered Arts](https://www.engineeredarts.co.uk/zh/%e9%a6%96%e9%a1%b5/))也都在研究長的更像人類的機器人，相信未來的發展還是很值得期待的

**Comment by SamTU0222** [comment]
謝謝分享 這樣好像會讓人們越來越依靠機器人

**Comment by handsomeniko** [comment]
恐怖谷真的很重要 我就超怕湯瑪士

**Comment by ThomasMeng0515** [comment]
Musk的想法真的都很新潮很有創意，值得學習

**Comment by d26488399** [comment]
每個人家裡都有機器人的話，感覺隱私問題會是一個隱憂

**Reply by Sazmafuji** [reply]
你旁邊的那(apple)方形機器人，你可能也需要擔憂一下

**Comment by Jay-722** [comment]
拜託幫忙智慧化打掃，該丟的丟

**Comment by bloodtrail0511** [comment]
希望這樣的構想可以趕快到來~

**Comment by jcpan821** [comment]
這種感覺就像是50年前沒人想的到現在每個人家裡都有彩色電視

**Comment by yucheng1028** [comment]
wc 底特律變人要成真了嗎

**Comment by henrywen547** [comment]
確實是有可能發生的，畢竟機器人真的有助人類減少任務量
